R/code-of-conduct.R

```diff
@@ -10,8 +10,8 @@
 #' If your package is going to CRAN, the link to the CoC in your README must
 #' be an absolute link to a rendered website as `CODE_OF_CONDUCT.md` is not
 #' included in the package sent to CRAN. `use_code_of_conduct()` will
-#' automatically generate this link if you use pkgdown and have set the
-#' `url` field in `pkgdown.yml`; otherwise it'll link to
+#' automatically generate this link if (1) you use pkgdown and (2) have set the
+#' `url` field in `_pkgdown.yml`; otherwise it'll link to
```

Comment:
If we're going to be specific, it should at least be the most common config filename.